frontend/src/components/auth/Credentials.tsx

```diff
@@ -92,65 +100,120 @@ const Credentials = ({
-              if ("tokenId" in response) {
-                onGoogleLoginSuccess(response.tokenId);
-              } else {
-                // eslint-disable-next-line no-alert
-                window.alert(response);
-              }
-            }}
-            onFailure={(error: GoogleErrorResponse) =>
-              // eslint-disable-next-line no-alert
-              window.alert(JSON.stringify(error))
-            }
-          />
-        </form>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onSignUpClick}
+      <Flex h="100vh">
+        <Box w="47%">
+          <Flex
+            position="absolute"
+            top="27%"
+            left="6%"
+            w="36%"
+            h="30%"
+            justifyContent="space-between"
+            direction="column"
           >
-            Sign Up
-          </button>
-        </div>
-      </div>
-    );
+            <Box display="flex"  alignItems="flex-start">
+              <Text 
+                position="absolute" 
+                fontWeight="700" 
+                fontSize="40px" 
+                color="#000000"
+              >
+                Log In
+              </Text>
+            </Box>
+            <Box>
+              <Input 
+                height="7vh" 
+                placeholder="Your email address"
+                fontWeight="400"
+                fontSize="22px"
+                lineHeight="29px"
+                position="absolute"
+                color="#989898"
+                _placeholder={{
+                color: "#989898"
+                }}
+                type="email"
+                value={email}
+                onChange={(event) => setEmail(event.target.value)}
+              />
+            </Box>
+            <Box>
+              <Input 
+                  height="7vh" 
+                  placeholder="Your password"
+                  fontWeight="400"
+                  fontSize="22px"
+                  lineHeight="29px"
+                  position="absolute"
+                  color="#989898"
+                  _placeholder={{
+                  color: "#989898", 
+                  }}
+                  type="password"
+                  value={password}
+                  onChange={(event) => setPassword(event.target.value)}
+                />
+            </Box>
+            <Box>
+              <Button
+                height="7vh"
+                width="100%"
+                bg="#285E61"
+                borderRadius="4px"
+                position="absolute"
+                fontWeight="400"
+                fontSize="22px"
+                lineHeight="29px"
+                color="#FFF"
+                _hover={{cursor: "default"}}
+                _focus={{ outline: "none", boxShadow: "none" }}
+                disabled={email === '' || password === ''}
+                type="button"
+                onClick={onLogInClick}
+              >
```

Comment:
if you look at `supportive-housing/frontend/src/theme/common/buttonStyles.tsx` you'll see our design system and the button stylings we have on there. I think you should be able to design this styling as a new variant and then you can use that styling here

Reply:
^^^ for the button styling you will want to use the values from `supportive-housing/frontend/src/theme/colors.tsx`. If you want to see the colours there is a figma page for it [here](https://www.figma.com/file/6Q5yT8sRQbv6VUR8hoj8ET/SHOW-Chakra-UI?type=design&node-id=1-2&mode=design&t=cVYqY9DXh6pmy77a-0)